faker/providers/person/__init__.py

```diff
@@ -66,6 +72,18 @@ def last_name_female(cls):
         return cls.last_name()
 
     @classmethod
+    def middle_name_male(cls):
```

Comment:
Not every provider supports middle names. You can move this logic to the localized russian subclass (`person/ru_RU/__init__.py`)